siege_utilities/survey/render.py

```
@@ -0,0 +1,199 @@
+        else:
+            df.plot.bar(ax=ax)
+    except Exception:
+        plt.close(fig)
+        return None
+
+    ax.set_title(headline, fontsize=12, fontweight="bold")
+    plt.tight_layout()
+    return fig
+
+
+def _build_map(chain: "Chain") -> Optional[Any]:
```

Ruff (0.15.10)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)